packages/cli/templates/csharp/echo/{{name}}.hbs/Program.cs.hbs

```diff
@@ -1,13 +1,19 @@
 using Microsoft.Teams.Plugins.AspNetCore.DevTools.Extensions;
 using Microsoft.Teams.Plugins.AspNetCore.Extensions;
 
-using Echo;
+using {{name}};
 
 var builder = WebApplication.CreateBuilder(args);
 builder.AddTeams();
 builder.AddTeamsDevTools();
 builder.Services.AddTransient<MainController>();
 
 var app = builder.Build();
+
+if (app.Environment.IsDevelopment())
+{
+    app.MapOpenApi();
```

Comment:
should we also only add devtools when in development?

Reply:
gonna do this in a ff due to time constraints